src/PeclEvLoop.php

```diff
@@ -0,0 +1,231 @@
+<?php
+
+namespace React\EventLoop;
+
+use Ev;
+use EvLoop;
+use React\EventLoop\Tick\FutureTickQueue;
+use React\EventLoop\Timer\Timer;
+use React\EventLoop\Timer\TimerInterface;
+use SplObjectStorage;
+
+/**
+ * @see https://bitbucket.org/osmanov/pecl-ev/overview
+ */
+class PeclEvLoop implements LoopInterface
+{
+    private $loop;
+    private $futureTickQueue;
+    private $timers;
+    private $readStreams = [];
+    private $writeStreams = [];
+    private $running;
+
+    public function __construct()
+    {
+        $this->loop             = new EvLoop();
+        $this->futureTickQueue  = new FutureTickQueue($this);
+        $this->timers           = new SplObjectStorage();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addReadStream($stream, callable $listener)
```

Comment:
Since this methods implements the interface, it's explicit that the docblock is inherited. We tend to avoid docblocks since they are often redundant and become easily out of date. With type hints and return types in PHP 7, they will become even more obsolete (at least 99.9% of the time) :)